app/helpers/profiles_helper.rb

```diff
@@ -0,0 +1,43 @@
+module ProfilesHelper
+  def likes_counter(likeable)
+    count = likeable.likes.count
+
+    if count == 0
+      str = link_to "Be the first to like this.", post_likes_path(likeable), method: :post
```

Comment:
I did not think about having a default message if nobody has liked a specific thing.